src/prettytable/prettytable.py

```diff
@@ -2693,7 +2696,7 @@ def from_html(html_code, **kwargs):
     return parser.tables
 
 
-def from_html_one(html_code, **kwargs):
+def from_html_one(html_code: str, **kwargs) -> PrettyTable:
     """
     Generates a PrettyTables from a string of HTML code which contains only a
```

Comment:
Just spotted this little typo, let's slip a fix in:

```suggestion
    Generates a PrettyTable from a string of HTML code which contains only a
```